includes/admin.php

```diff
@@ -255,27 +257,31 @@ function rns_transmissions_lists_available_cboxes() {
 	echo '<legend class="screen-reader-text">Available lists</legend>';
 	echo '<div id="rns-default-lists-groups">';
 
-	foreach ( $available_lists as $name => $ID ) {
-		$disabled = ( empty( $available_groups[$ID] ) ) ? 'disabled="disabled"' : '';
-		echo '<label for="rns_transmissions_options_' . $ID . '">';
-		echo '<input ' . $disabled . ' ' . checked( $ID, $default_list, false ) . ' name="rns_transmissions_options[lists_enabled]" type="radio" id="rns_transmissions_options_' . $ID . '" value="' . $ID . '"> ';
-		echo $name;
-
-		if ( ! empty( $available_groups[$ID] ) ) {
-			$default_group = $options['list_groups'][$ID]['default_group'];
-
-			echo '<br>';
-			echo '<div class="nested-group" style="margin: 10px 0 0 20px;">';
-			foreach ( $available_groups[$ID] as $group ) {
-				echo '<label for="rns_transmissions_options_'. $group['id'] . '_default_group">';
-				echo '<input ' . checked( $group['id'], $default_group, false ) . ' name="rns_transmissions_options[list_groups][' . $ID . '][default_group]" type="radio" id="rns_transmissions_options_' . $group['id'] . '_default_group" value="' . $group['id'] . '" >';
-				echo $group['name'];
-				echo '</label><br>';
+	if ( $available_lists ) {
+		foreach ( $available_lists as $name => $id ) {
+			$disabled = ( empty( $available_groups[ $id ] ) ) ? 'disabled="disabled"' : '';
+			echo '<label for="rns_transmissions_options_' . intval( $id ) . '">';
+			echo '<input ' . esc_attr( $disabled ) . ' ' . checked( $id, $default_list, false ) . ' name="rns_transmissions_options[lists_enabled]" type="radio" id="rns_transmissions_options_' . intval( $id ) . '" value="' . intval( $id ) . '"> ';
+			echo esc_html( $name );
+
+			if ( ! empty( $available_groups[ $id ] ) ) {
+				$default_group = $options['list_groups'][ $id ]['default_group'];
+
+				echo '<br>';
+				echo '<div class="nested-group" style="margin: 10px 0 0 20px;">';
+				foreach ( $available_groups[ $id ]['categories'] as $group ) {
+					echo '<label for="rns_transmissions_options_' . intval( $group['id'] ) . '_default_group">';
+					echo '<input ' . checked( $group['id'], $default_group, false ) . ' name="rns_transmissions_options[list_groups][' . intval( $id ) . '][default_group]" type="radio" id="rns_transmissions_options_' . intval( $group['id'] ) . '_default_group" value="' . intval( $group['id'] ) . '" >';
+					echo esc_html( $group['title'] );
+					echo '</label><br>';
+				}
+				echo '</div>';
```

Comment:
This whole section was just an indentation change and not any functionality change, correct?